docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
```

Comment:
```suggestion
How to manage your Read the Docs for Business subscription
==========================================================
```

Reply:
This page isn't included in a TOC, but once included, it can have a different label in the TOC.

Reply:
Ah, it's implicitly in a TOC? Well, we can probably follow up on the title in any case.